webdriver-spec.html

```diff
@@ -3884,6 +3884,16 @@
 
  <li><p>Let <var>selector</var> be equal to <var>value</var>.
 
+ <li><p>If <var>selector</var> is equal to <code>""</code> or <a>null</a>, match on <var>all</var>:
```

Comment:
If the selector is an empty string or null it is an invalid selector, which will throw a DOM `SyntaxError` error (which we currently don’t handle).

It seems strange to special case a certain subclass of invalid selectors like this.